tools/accuracy_checker/accuracy_checker/utils.py

```diff
@@ -20,16 +20,32 @@
 import json
 import os
 import pickle
+import struct
+import sys
+import zlib
 from enum import Enum
 
 from pathlib import Path
 from typing import Union
 from warnings import warn
 from collections.abc import MutableSet
 
+try:
+    from collections.abc import Sequence
+except ImportError:
+    from collections import Sequence
```

Comment:
if I right understand, try-except here need for compatibility with python 2.7, but our target is python3.5 and higher, probably we can use only first statement.

Reply:
and look above, we already has import from `collections.abc` :)